DeAceroDLabs/marketplace-frontend
# [DMP-558] Get user data from BE

## What is done

[[DMP-558] Obtener datos de Usuario desde el BE](https://deacero.atlassian.net/browse/DMP-558)

## How is it done

1. Get user data using the get users by email endpoint.
2. Store user data on local storage to be always available.
3. Get user data in the Payment component from the local storage.

## How to test

1. Run the project via `npm start`
2. Login to the app.
3. Go to `/carrito/pago`
4. Check that the location info from your user appears correct.
